packages/select/package.json

```diff
@@ -56,7 +56,8 @@
     "uuid": "^8.3.2"
   },
   "devDependencies": {
-    "rimraf": "^3.0.2"
+    "rimraf": "^3.0.2",
+    "@tanstack/react-query": "^4.12.0"
```

Comment:
Given you've added the react-query stuff to the top level storybook preview, this would make make more sense in the root `package.json` now

Reply:
Updated.